astrobee/launch/robot/MLP.launch.py

```diff
@@ -59,13 +59,12 @@ def generate_launch_description():
             #     plugin='localization_manager::LocalizationManagerNodelet',
             #     name='localization_manager',
             #     extra_arguments=[{'use_intra_process_comms': True}]),
-            # ComposableNode(
-            #     package='ground_truth_localizer',
-            #     plugin='ground_truth_localizer::GroundTruthLocalizerNodelet',
-            #     name='ground_truth_localizer',
-            #     remappings=[('/image', '/burgerimage')],
-            #     parameters=[{'history': 'keep_last'}],
-            #     extra_arguments=[{'use_intra_process_comms': True}]),
+            ComposableNode(
+                package='ground_truth_localizer',
+                plugin='ground_truth_localizer::GroundTruthLocalizerNodelet',
+                name='ground_truth_localizer',
+                # condition=IfCondition(LaunchConfiguration("gtloc"))
```

Comment:
So the `ComposableNode` doesn't support the `condition` arg, will open a PR trying to solve this separately -.-